analytics/management/commands/populate_analytics_db.py

```diff
@@ -63,8 +63,7 @@ def handle(self, *args, **options):
 
         def insert_fixture_data(stat, fixture_data, table):
             # type: (CountStat, Dict[Optional[str], List[int]], Type[BaseCount]) -> None
-            end_times = time_range(last_end_time, last_end_time, stat.frequency,
-                                   len(list(fixture_data.values())[0]))
+            end_times = time_range(last_end_time, last_end_time, stat.frequency)
```

Comment:
This will no longer work (you can test by running `./manage.py populate_analytics_db`. Okay, let's instead leave `min_length` as an argument to `time_range`, but just remove it as an argument to `get_chart_data`.